vladiate/validators.py

```diff
@@ -208,3 +210,14 @@ def validate(self, field, row={}):
     @property
     def bad(self):
         pass
+
+
+def stringify_set(a_set, max_len):
+    ''' Stringify `max_len` elements of `a_set` and count the remainings '''
+    # Don't convert `a_set` to a list for performance reasons
+    text = "[{}]".format(", ".join(
```

Comment:
Small tweak, could you change this to `"[{}]"` to `"{{{}}}"` so that this prints the set in braces instead of brackets and is consistent with the way it was before?

Reply:
Sure, that makes sense.